rlkit/torch/networks/cnn.py

```diff
@@ -0,0 +1,350 @@
+import numpy as np
+import torch
+from torch import nn as nn
+
+from rlkit.policies.base import Policy
+from rlkit.pythonplusplus import identity
+from rlkit.torch.core import PyTorchModule, eval_np
+from rlkit.torch.data_management.normalizer import TorchFixedNormalizer
+
+
+from rlkit.torch.pytorch_util import activation_from_string
+
+
+class CNN(PyTorchModule):
```

Comment:
btw, are you still using this? it could be nice to only use `BasicCNN`

Reply:
It's in a few imports but I am not really using this in recent code. The only file that does seem to use that might be a bit annoying to address is ConvVAE (because a lot of other code uses it). So I think deleting this is best deferred to later?